### ISSUE TITLE
Fake Tensor Enablement Follow - Ups

### ISSUE THREAD

**Issue by eellison** [pytorch/torchdynamo#1215]
- fix batchnorm 2d decomp cpu error 
(More to be filled)
